packages/plugins/chainflip/src/plugin.ts

```diff
@@ -76,6 +81,7 @@ function plugin({
       buyAsset,
       recipient,
       sellAsset,
+      maxBoostFeeBps: maxBoostFeeBps ?? 0,
```

Comment:
This condition never "fires" as `maxBoostFeeBps` is required param.

Reply:
done